test.go

```diff
@@ -201,6 +201,9 @@ func (t *Test) run(dir, dataDir string) ([]byte, error) {
 		} else {
 			// If there is no cache, we have to call 'make build' before each test.
 			args = []string{t.flavor, t.bootstrapVersion, "make build && " + testArgs}
+			if !*buildVTAdmin {
+				args[2] = "NOVTADMINBUILD=1 " + args[2]
```

Comment:
We might want to check the len of args, or append to args[len-1] instead. But up to you.

Reply:
Resolved via e5c394236c